.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/python/black
-    rev: 19.3b0
+    rev: 19.10b0
```

Comment:
Latest is 20.8b1 of black. Should we upgrade to that?

Reply:
Good point. In addition, I made a few changes to how we install black and sync appveyor as well. Waiting for AppVeyor's check.